charts/opentelemetry-operator/values.yaml

```diff
@@ -130,6 +130,14 @@ manager:
   # Enable manager pod automatically rolling
   rolling: false
 
+  ## Container specific securityContext
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  securityContext:
```

Comment:
I don't think we need to provide any default values for with securityContext section. Lets set them to `securityContext: {}`.